### PR TITLE
New package: Google.Android.SDKPlatformTools version 33.0.3

### LINKED ISSUE #4082: Android SDK Platform-Tools
# Package Requested

Manifest:

Fill out as much of the manifest as you can:

```YAML
Id: string # publisher.package format
Publisher: Google # the name of the publisher
Name: Android SDK Platform-Tools # the name of the application
Version: 30.0.4 # version numbering format
License: developer.android.com/license # the open source license or copyright
InstallerType: exe (standalone, no installer) # enumeration of supported installer types (exe, msi, msix, inno, wix, nullsoft, appx)
Installers:
  - Arch: string # enumeration of supported architectures
    Url: https://dl.google.com/android/repository/platform-tools_r30.0.4-windows.zip # path to download installation file
    Sha256: string # SHA256 calculated from installer
# ManifestVersion: 0.1.0
```

### PR DESCRIPTION
**Pull request by sitiom**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

* Resolves #4082

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94818)

### CODE REVIEW

**Comment by sitiom** [comment]
Suffers the same problem with https://github.com/microsoft/winget-pkgs/pull/92418. Cannot find DLLs due to symlinking. `adb` silently exits with no output.

**Comment by sitiom** [comment]
a

**Comment by sitiom** [comment]
Blocked by https://github.com/microsoft/winget-cli/issues/2889

File: manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PublisherSupportUrl: https://developer.android.com/studio/intro
+PrivacyUrl: https://policies.google.com/privacy
+# Author:
+PackageName: Android SDK Platform-Tools
+PackageUrl: https://developer.android.com/studio/releases/platform-tools
+License: Android Software Development Kit License Agreement
+LicenseUrl: https://developer.android.com/studio/terms
+Copyright: Copyright (c) Google LLC.
+CopyrightUrl: https://developer.android.com/studio/terms
+ShortDescription: Android SDK platform-tools includes tools that interface with the Android platform, such as adb, fastboot, and systrace
```
Comment (DanielHabenicht) [inline manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Android SDK platform-tools includes tools that interface with the Android platform, such as adb and fastboot.
```
>  This package used to contain systrace, but that has been obsoleted in favor of Studio Profiler, gpuinspector.dev, or Perfetto.
https://developer.android.com/studio/releases/platform-tools

File: manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.Android.SDKPlatformTools
+PackageVersion: 33.0.3
+InstallerType: zip
+Installers:
```
Comment (DanielHabenicht) [inline manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.installer.yaml:7]:
```suggestion
Commands:
- adb
- fastboot
- dmtracedump
- etc1tool
- hprof-conv
Installers:
```
When I created the Package locally (didn't see your PR then) it added this Commands section. 
Maybe its needed? (but I am not that familar with WinGet)

Comment (sitiom) [reply manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.installer.yaml:7]: This field is ignored when `InstallerType` is a zip.

Comment (DanielHabenicht) [reply manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.installer.yaml:7]: Alright, nice to know.

**Review by ItzLevvie — APPROVED** [review]

**Review by Gdakgdak22 — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
@sitiom - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Comment by sitiom** [comment]
Continued in #190804

**Pull request closed without merging**